Makefile

```diff
@@ -290,3 +295,35 @@ $(GOPATH)/bin/gomod2rpmdeps:
 	@sed -e 's/__VERSION__/'$(CRC_VERSION)'/g' \
 	     -e 's/__OPENSHIFT_VERSION__/'$(BUNDLE_VERSION)'/g' \
 	     $< >$@
+
+$(HOST_BUILD_DIR)/split: packaging/windows/split.go
+	go build -o $(HOST_BUILD_DIR)/split packaging/windows/split.go
+
+$(PACKAGE_DIR)/product.wxs: $(PACKAGE_DIR)/product.wxs.in
```

Comment:
This specific rule should not be needed, from `msidir: $(PACKAGE_DIR)/product.wxs`, `make` should realize automatically it needs to use the `%.in:%` rule.